apis/infrastructure/v1beta1/byohost_webhook.go

```diff
@@ -20,23 +22,55 @@ type ByoHostValidator struct {
 	decoder *admission.Decoder
 }
 
+// To allow byoh manager service account to patch ByoHost CR
+const managerServiceAccount = "system:serviceaccount:byoh-system:byoh-controller-manager"
+
 // nolint: gocritic
```

Comment:
Any reason to add `nolint` here?

Reply:
This was already there, but I will check what its required for.

Reply:
This is there for below lint issue, which couldn't be worked upon as this is an implementation of `admission.Handler` Interface 
`apis/infrastructure/v1beta1/byohost_webhook.go:29:56: hugeParam: req is heavy (400 bytes); consider passing it by pointer (gocritic)`